<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTeamWindow</class>
 <widget class="QMainWindow" name="CreateTeamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Team</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>531</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0F111A;
color: #FFFFFF;
font-family: 'Segoe UI', Arial, sans-serif;
font-size: 12px;
border-radius: 15px;
padding: 15px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_title">
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; font-size: 18px; font-weight: bold; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Create Your Team</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_team_name">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Team Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="teamNameEdit">
       <property name="styleSheet">
        <string notr="true">background-color: #1C1F2A; border: 1px solid #2A2D3A; border-radius: 5px; padding: 5px; color: white;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_home_stadium">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Home Stadium</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="stadiumEdit">
       <property name="styleSheet">
        <string notr="true">background-color: #1C1F2A; border: 1px solid #2A2D3A; border-radius: 5px; padding: 5px; color: white;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_goals">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Goals</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="goalsEdit">
       <property name="styleSheet">
        <string notr="true">background-color: #1C1F2A; border: 1px solid #2A2D3A; border-radius: 5px; padding: 5px; color: white;</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_budget">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Budget</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="budgetEdit">
       <property name="styleSheet">
        <string notr="true">background-color: #1C1F2A; border: 1px solid #2A2D3A; border-radius: 5px; padding: 5px; color: white;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_team_logo">
       <property name="styleSheet">
        <string notr="true">color: #A0A8B7; font-weight: bold; padding-bottom: 5px; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Team Logo (File Path)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLineEdit" name="teamLogoEdit">
       <property name="styleSheet">
        <string notr="true">background-color: #1C1F2A; border: 1px solid #2A2D3A; border-radius: 5px; padding: 5px; color: white;</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="createButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(50, 150, 250, 0.8); color: white; font-size: 12px; border-radius: 5px; padding: 6px 12px;</string>
       </property>
       <property name="text">
        <string>Create Team</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>